src/main/java/ignatProject/service/lmpl/BasketServiceImpl.java

```diff
@@ -0,0 +1,77 @@
+package ignatProject.service.lmpl;
+
+import ignatProject.bean.Basket;
+import ignatProject.service.BasketService;
+import org.springframework.stereotype.Service;
+import ignatProject.repository.BasketRepository;
+import ignatProject.repository.BookRepository;
+
+import java.util.List;
+
+import static org.aspectj.bridge.MessageUtil.info;
+
+@Service
+public class BasketServiceImpl implements BasketService {
+
+    final
+    BookRepository bookRepository;
```

Comment:
final BookRepository bookRepository` change to 
```
@Autowired
protected BookRepository bookRepository
```
and remove constructor in this class

Reply:
the similar changes needed to be done in other service classes